src/runtime/node/constants/index.ts

```diff
@@ -0,0 +1,16 @@
+// Mostly taken from: https://github.com/nodejs/node/blob/main/typings/internalBinding/constants.d.ts
+
+import constants from "node:constants";
```

Comment:
did you mean `type`?

```suggestion
import type constants from "node:constants";
```